docker compose logs -f backend
[2Kbackend-1  | No changes detected
[2Kbackend-1  | Operations to perform:
[2Kbackend-1  |   Apply all migrations: admin, auth, backendApi, contenttypes, sessions
[2Kbackend-1  | Running migrations:
[2Kbackend-1  |   Applying contenttypes.0001_initial... OK
[2Kbackend-1  |   Applying contenttypes.0002_remove_content_type_name... OK
[2Kbackend-1  |   Applying auth.0001_initial... OK
[2Kbackend-1  |   Applying auth.0002_alter_permission_name_max_length... OK
[2Kbackend-1  |   Applying auth.0003_alter_user_email_max_length... OK
[2Kbackend-1  |   Applying auth.0004_alter_user_username_opts... OK
[2Kbackend-1  |   Applying auth.0005_alter_user_last_login_null... OK
[2Kbackend-1  |   Applying auth.0006_require_contenttypes_0002... OK
[2Kbackend-1  |   Applying auth.0007_alter_validators_add_error_messages... OK
[2Kbackend-1  |   Applying auth.0008_alter_user_username_max_length... OK
[2Kbackend-1  |   Applying auth.0009_alter_user_last_name_max_length... OK
[2Kbackend-1  |   Applying auth.0010_alter_group_name_max_length... OK
[2Kbackend-1  |   Applying auth.0011_update_proxy_permissions... OK
[2Kbackend-1  |   Applying auth.0012_alter_user_first_name_max_length... OK
[2Kbackend-1  |   Applying backendApi.0001_initial... OK
[2Kbackend-1  |   Applying admin.0001_initial... OK
[2Kbackend-1  |   Applying admin.0002_logentry_remove_auto_add... OK
[2Kbackend-1  |   Applying admin.0003_logentry_add_action_flag_choices... OK
[2Kbackend-1  |   Applying backendApi.0002_remove_channel_banneds_remove_channel_inviteds_and_more... OK
[2Kbackend-1  |   Applying backendApi.0003_alter_channel_name... OK
[2Kbackend-1  |   Applying backendApi.0004_alter_channel_name... OK
[2Kbackend-1  |   Applying backendApi.0005_alter_banneduser_receiver_alter_banneduser_sender_and_more... OK
[2Kbackend-1  |   Applying backendApi.0006_remove_friendship_unique_friendship_and_more... OK
[2Kbackend-1  |   Applying backendApi.0007_channelbanneduser_bannedreason_and_more... OK
[2Kbackend-1  |   Applying backendApi.0008_remove_game_loser_remove_game_loserscore_and_more... OK
[2Kbackend-1  |   Applying backendApi.0009_alter_tournament_status... OK
[2Kbackend-1  |   Applying backendApi.0010_alter_tournament_max_players_and_more... OK
[2Kbackend-1  |   Applying backendApi.0011_alter_user_created_at_alter_user_updated_at... OK
[2Kbackend-1  |   Applying backendApi.0012_alter_tournament_end_date_and_more... OK
[2Kbackend-1  |   Applying backendApi.0013_alter_user_level... OK
[2Kbackend-1  |   Applying backendApi.0014_notification... OK
[2Kbackend-1  |   Applying backendApi.0015_alter_notification_user... OK
[2Kbackend-1  |   Applying backendApi.0016_alter_user_avatarpath... OK
[2Kbackend-1  |   Applying sessions.0001_initial... OK
[2Kbackend-1  | Watching for file changes with StatReloader
[2Kbackend-1  | [12/May/2024 13:43:12] "OPTIONS /api/v1/profile/me/avatar HTTP/1.1" 200 0
[2Kbackend-1  | Forbidden: /api/v1/profile/me/avatar
[2Kbackend-1  | [12/May/2024 13:43:12] "GET /api/v1/profile/me/avatar HTTP/1.1" 403 26
[2Kbackend-1  | [12/May/2024 13:45:37] "OPTIONS /api/v1/auth/register HTTP/1.1" 200 0
[2Kbackend-1  | [12/May/2024 13:45:37] "POST /api/v1/auth/register HTTP/1.1" 201 36
[2Kbackend-1  | [12/May/2024 13:45:41] "POST /api/v1/auth/otp/status HTTP/1.1" 200 19
[2Kbackend-1  | [12/May/2024 13:45:42] "POST /api/v1/auth/login HTTP/1.1" 200 514
[2Kbackend-1  | [12/May/2024 13:45:42] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:45:42] "GET /api/v1/user/friendship HTTP/1.1" 200 2
[2Kbackend-1  | [12/May/2024 13:45:42] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:45:42] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:45:42] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | [12/May/2024 13:45:42] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | Internal Server Error: /api/v1/auth/logout
[2Kbackend-1  | Traceback (most recent call last):
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[2Kbackend-1  |     response = get_response(request)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
[2Kbackend-1  |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
[2Kbackend-1  |     return view_func(request, *args, **kwargs)
[2Kbackend-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/viewsets.py", line 124, in view
[2Kbackend-1  |     return self.dispatch(request, *args, **kwargs)
[2Kbackend-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
[2Kbackend-1  |     response = self.handle_exception(exc)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
[2Kbackend-1  |     self.raise_uncaught_exception(exc)
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
[2Kbackend-1  |     raise exc
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
[2Kbackend-1  |     response = handler(request, *args, **kwargs)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/home/backend/src/backendApi/views/user.py", line 77, in logOut
[2Kbackend-1  |     jwtTokenKeys = request.session.get("jwtToken", None).get(user.username, None)
[2Kbackend-1  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  | AttributeError: 'NoneType' object has no attribute 'get'
[2Kbackend-1  | [12/May/2024 13:45:44] "POST /api/v1/auth/logout HTTP/1.1" 500 93157
[2Kbackend-1  | [12/May/2024 13:45:44] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:45:44] "GET /api/v1/profile/me HTTP/1.1" 200 320
[2Kbackend-1  | [12/May/2024 13:48:15] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:48:21] "POST /api/v1/auth/otp/status HTTP/1.1" 200 19
[2Kbackend-1  | [12/May/2024 13:48:22] "POST /api/v1/auth/login HTTP/1.1" 200 514
[2Kbackend-1  | [12/May/2024 13:48:22] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:48:22] "GET /api/v1/user/friendship HTTP/1.1" 200 2
[2Kbackend-1  | [12/May/2024 13:48:22] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:48:22] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:48:22] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | [12/May/2024 13:48:22] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | [12/May/2024 13:51:01] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:01] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:01] "GET /api/v1/user/friendship HTTP/1.1" 200 2
[2Kbackend-1  | [12/May/2024 13:51:01] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:01] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:01] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | [12/May/2024 13:51:01] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | [12/May/2024 13:51:01] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | Internal Server Error: /api/v1/auth/logout
[2Kbackend-1  | Traceback (most recent call last):
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[2Kbackend-1  |     response = get_response(request)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
[2Kbackend-1  |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
[2Kbackend-1  |     return view_func(request, *args, **kwargs)
[2Kbackend-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/viewsets.py", line 124, in view
[2Kbackend-1  |     return self.dispatch(request, *args, **kwargs)
[2Kbackend-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
[2Kbackend-1  |     response = self.handle_exception(exc)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
[2Kbackend-1  |     self.raise_uncaught_exception(exc)
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
[2Kbackend-1  |     raise exc
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
[2Kbackend-1  |     response = handler(request, *args, **kwargs)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/home/backend/src/backendApi/views/user.py", line 77, in logOut
[2Kbackend-1  |     jwtTokenKeys = request.session.get("jwtToken", None).get(user.username, None)
[2Kbackend-1  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  | AttributeError: 'NoneType' object has no attribute 'get'
[2Kbackend-1  | [12/May/2024 13:51:04] "POST /api/v1/auth/logout HTTP/1.1" 500 93157
[2Kbackend-1  | [12/May/2024 13:51:04] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:04] "GET /api/v1/profile/me HTTP/1.1" 200 320
[2Kbackend-1  | [12/May/2024 13:51:06] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:13] "POST /api/v1/auth/otp/status HTTP/1.1" 200 19
[2Kbackend-1  | [12/May/2024 13:51:14] "POST /api/v1/auth/login HTTP/1.1" 200 514
[2Kbackend-1  | [12/May/2024 13:51:14] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:14] "GET /api/v1/user/friendship HTTP/1.1" 200 2
[2Kbackend-1  | [12/May/2024 13:51:14] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:14] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:14] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | [12/May/2024 13:51:14] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | Internal Server Error: /api/v1/auth/logout
[2Kbackend-1  | Traceback (most recent call last):
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
[2Kbackend-1  |     response = get_response(request)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
[2Kbackend-1  |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
[2Kbackend-1  |     return view_func(request, *args, **kwargs)
[2Kbackend-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/viewsets.py", line 124, in view
[2Kbackend-1  |     return self.dispatch(request, *args, **kwargs)
[2Kbackend-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
[2Kbackend-1  |     response = self.handle_exception(exc)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
[2Kbackend-1  |     self.raise_uncaught_exception(exc)
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
[2Kbackend-1  |     raise exc
[2Kbackend-1  |   File "/root/.local/share/virtualenvs/src-_NrZQeoR/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
[2Kbackend-1  |     response = handler(request, *args, **kwargs)
[2Kbackend-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  |   File "/home/backend/src/backendApi/views/user.py", line 77, in logOut
[2Kbackend-1  |     jwtTokenKeys = request.session.get("jwtToken", None).get(user.username, None)
[2Kbackend-1  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2Kbackend-1  | AttributeError: 'NoneType' object has no attribute 'get'
[2Kbackend-1  | [12/May/2024 13:51:47] "POST /api/v1/auth/logout HTTP/1.1" 500 93157
[2Kbackend-1  | [12/May/2024 13:51:47] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:51:47] "GET /api/v1/profile/me HTTP/1.1" 200 320
[2Kbackend-1  | [12/May/2024 13:54:25] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:54:31] "POST /api/v1/auth/otp/status HTTP/1.1" 200 19
[2Kbackend-1  | [12/May/2024 13:54:32] "POST /api/v1/auth/login HTTP/1.1" 200 514
[2Kbackend-1  | [12/May/2024 13:54:32] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:54:32] "GET /api/v1/user/friendship HTTP/1.1" 200 2
[2Kbackend-1  | [12/May/2024 13:54:32] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:54:32] "GET /api/v1/profile/me HTTP/1.1" 200 319
[2Kbackend-1  | [12/May/2024 13:54:32] "GET /api/v1/profile/me/avatar HTTP/1.1" 200 22381
[2Kbackend-1  | [12/May/2024 13:54:32] "GET /api/v1/profile/me HTTP/1.1" 200 319
